src/main/java/org/jabref/gui/groups/GroupDialogViewModel.java

```diff
@@ -212,8 +214,27 @@ private void setupValidation() {
                 input -> !StringUtil.isNullOrEmpty(input),
                 ValidationMessage.error(String.format("%s > %n %s",
                         Localization.lang("Free search expression"),
-                        Localization.lang("Search term is empty.")
-                )));
+                        Localization.lang("Search term is empty."))));
+
+        texGroupFilePathValidator = new FunctionBasedValidator<>(
+                texGroupFilePathProperty,
+                input -> {
+                    if (StringUtil.isNullOrEmpty(input)) {
+                        return false;
+                    } else if (input.trim().length() == 0) {
+                        return false;
+                    } else if ((input.length() < 4) || !input.substring(input.length() - 4).toLowerCase().equals(".aux")) {
+                        return false;
+                    } else {
+                        Path path = preferencesService.getWorkingDir();
+                        File texFile = new File(path.toString(), input);
+                        if (!texFile.exists() || !texFile.isFile()) {
+                            return false;
+                        }
+                        return true;
```

Comment:
Have you tried `FileUtil.find()` here?

Reply:
I'm not sure, whether this is a good idea. I intended to explicitly check the provided file, without searching other locations. :thinking:

Reply:
I have adapted it.